packages/kbn-grid-layout/grid/grid_panel/grid_panel.tsx

```diff
@@ -77,14 +77,6 @@ export const GridPanel = forwardRef<HTMLDivElement, GridPanelProps>(
         grid-column-end: ${initialPanel.column + 1 + initialPanel.width};
         grid-row-start: ${initialPanel.row + 1};
         grid-row-end: ${initialPanel.row + 1 + initialPanel.height};
-        &.kbnGridPanel--isExpanded {
```

Comment:
All styles related to expanding a panel were moved to the `GridLayout` component so that it is easier to keep track of them.